sgx-types/src/tcs.rs

```diff
@@ -1,10 +1,9 @@
-const SGX_TCS_RESERVED_SIZE: usize = 4024;
-
 /// Thread Control Structure (TCS) is an enclave page visible in its address
 /// space that defines an entry point inside the enclave. A thread enters inside
 /// an enclave by supplying address of TCS to ENCLU(EENTER). A TCS can be entered
 /// by only one thread at a time.
-#[repr(C, align(4096))]
+#[derive(Debug, Default)]
```

Comment:
I'm happy to see that we can use this now that we don't need the large `reserved` field as padding.